server/app/api/org/org_service.py

```diff
@@ -137,6 +139,30 @@ def add_members_to_org(self, org_id: int, user_ids: list[int], owner_id: int):
             new_members.append(user)
         return new_members
 
+    def search_members_by_username_prefix(self, query: str, team_id_to_exclude_members: int) -> List[User]:
+        team = self.team_repo.get(team_id_to_exclude_members)
+        if not team:
+            raise NotFoundException("Team", team_id_to_exclude_members)
+        org_members = self.org_repo.search_members_by_username_prefix(query, team.organization_id)
+        members_to_exclude = self.team_repo.find_members_of_team(team_id_to_exclude_members)
+        filtered_users = [
+            u for u in org_members
+            if u.id != team.organization.owner_id and u not in members_to_exclude
+        ]
+        return filtered_users
+
+    def remove_org_member(self, member_id: int, org_id: int, user_id: int):
+        om = self.org_repo.find_member(org_id, member_id)
+        if om is None:
+            raise NotFoundException(OrganizationMembers, member_id)
+        org = self.org_repo.find_by_id(org_id)
+        if org.owner_id != user_id:
+            raise AccessDeniedException(f"User {user_id} cannot remove member {member_id} from organization {org_id}")
+        team_members = self.team_repo.find_teams_of_member(member_id)
+        for tm in team_members:
```

Comment:
User should not be able to remove himself out of the org i.e. `member_id != org.owner_id`  must be true. The button in that case should also be hidden from the frontend.